R/write.R

```diff
@@ -118,3 +137,22 @@ xportr_write <- function(.df,
 
   invisible(data)
 }
+
+#' Figure out path for split data.
+#'
+#' Will append a number before teh file extension to indicate the split.
+#'
+#' i.e. `adsl.xpt` will become `adsl1.xpt` and `adsl2.xpt`
```

Comment:
in `xportr_split` should we tell the user this is happening, i.e. the number being placed at end of dataset name?

Reply:
Good catch. I through that in the xportr_split docs